web/html/src/manager/minion/packages/package-utils.ts

```diff
@@ -1,14 +1,14 @@
-import { OptionalValue, Package } from "./package.type";
+import { Package } from "./package.type";
 
-export const UNMANAGED: any = {};
-export const INSTALLED: OptionalValue = { value: 0 };
-export const REMOVED: OptionalValue = { value: 1 };
-export const PURGED: OptionalValue = { value: 2 };
+export const UNMANAGED: any = undefined;
```

Comment:
I would recommend removing `any` here, that way we can actually see where this `undefined` can propagate and end up.